CMakeLists.txt

```diff
@@ -210,6 +217,7 @@ if(NOT WIN32)
         core_optional_dep(LibDRM ENABLE_MIR)
       else()
         core_optional_dep(X ENABLE_X11)
+        core_optional_dep(XRandR ENABLE_X11)
```

Comment:
@FernetMenta things are indeed getting more clear now, thanks, learning everyday. i now got it working by completely removing the AML dependency so only the X11 platform stuff is used without using egl native type EGLNativeTypeAmlogic anymore.